operator/controllers/syncer/csi_node.go

```diff
@@ -354,7 +355,7 @@ func (s *csiNodeSyncer) getEnvFor(name string) []corev1.EnvVar {
 		} else {
 			OpenShiftObj.Value = "False"
 		}
-		EnvVars = append(EnvVars, OpenShiftObj)
+		//EnvVars = append(EnvVars, OpenShiftObj)
```

Comment:
The above block of handling of ENVIsOpenShift also needs to be removed now

Reply:
done